src/main/java/org/entando/kubernetes/controller/app/JbossDatasourceValidation.java

```diff
@@ -0,0 +1,85 @@
+/*
+ *
+ * Copyright 2015-Present Entando Inc. (http://www.entando.com) All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ *  This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ */
+
+package org.entando.kubernetes.controller.app;
+
+/**
+ * contains the ValidConnectionChecker and ExceptionSorted class names.
+ */
+
+import java.util.Optional;
+import org.entando.kubernetes.model.DbmsVendor;
+
+public enum JbossDatasourceValidation {
```

Comment:
Is this only a Jboss thing or generic?

Reply:
Yes it is only Jboss related. The referenced classes are jboss internals